flink-core/src/main/java/org/apache/flink/core/fs/Path.java

```diff
@@ -217,7 +211,17 @@ public Path(String pathString) {
 	 *        the path string
 	 */
 	public Path(String scheme, String authority, String path) {
-		checkPathArg(path);
+
+		if(path == null) {
```

Comment:
I think it would be better to keep the check in a method as before. I think it's a good practice to trim the path :-)